exid-types.json

```diff
@@ -81,4 +81,28 @@
     "change-controller": "FindAGrave.com",
     "reference": "https://support.findagrave.com/s/article/Cemetery-Search"
   }
+  {
+    "label": "BillionGraves Grave ID",
+    "type": "https://www.billiongraves.com/grave",
+    "description": "BillionGraves Grave ID",
+    "contact": "GEDCOM@familysearch.org",
+    "change-controller": "BillionGraves.com",
+    "reference": "https://billiongraves.com/search"
+  },
+  {
+    "label": "BillionGraves Cemetery ID",
+    "type": "https://www.billiongraves.com/cemetery",
```

Comment:
```suggestion
    "type": "https://www.billiongraves.com/cemetery/",
```

Reply:
Likewise here, it would be https://www.billiongraves.com/cemetery/name/